Src/Plugins/Security/SEC101_036.MySqlCredentialsValidator.cs

```diff
@@ -72,7 +73,7 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(Dictionary<
             // Username must be in the form <username>@<hostname> to communicate with Azure.
             // If the username does not contain a host name, we can't connect.
             if (AzureHosts.Any(azHosts => hostValue.IndexOf(azHosts, StringComparison.OrdinalIgnoreCase) != -1) &&
```

Comment:
If we wanted to be consistent across our validators might want to change this to 

```csharp
AzureHosts.Any(azHost => hostValue.Contains(azHost, StringConmparison.OrdinalIgnoreCase))...
```

Functionally the same, easier (IMO) to read.

Reply:
Contains does not have stringcomparison

Reply:
the version  that contain 'contains' with StringComparison is net5, which does not apply here.